Cargo.toml

```diff
@@ -80,6 +80,9 @@ felt = { package = "cairo-felt", path = "./felt", version = "0.3.0-rc1", default
 ] }
 bitvec = { version = "1", default-features = false, features = ["alloc"] }
 
+[dev-dependencies]
+proptest = "1.0.0"
```

Comment:
I'm not sure `wasm32` actually supported this. I don't know why the dep was there.

Reply:
I saw that the felt crate had it as a dev-dependency, and used proptest unconditionally, while in cairo-vm we had the `#[cfg(not(target_arch = "wasm32"))]` before proptest, so I tried to make it work for cairo-vm like it does for felt. But wasm tests are now failling so Ill probably revert the change and see if I can fix it in a later PR

Reply:
Yes, that's why I'm asking. I remember seeing some `cfg(not(...))` around proptest at some point. But if it works all the better.